src/utils/text/pluralize.js

```diff
@@ -0,0 +1,19 @@
+class Pluralize {
+  static _pickRightForm(count, singular, plural) {
+    return count === 1 ? singular : plural;
+  }
+  static word(count) {
+    return this._pickRightForm(count, "word", "words");
```

Comment:
HAHA! You're fast @tfnribeiro .

here's an idea of further reducing duplication maybe... introducing the `_regular` helper function. 

```js  
static _regular(count, singular) {
    return this._pickRightForm(count, singular, singular + "s");
  }

  // Regular plurals
  static word(count) {
    return this._regular(count, "word");
  }

  static second(count) {
    return this._regular(count, "second");
  }

  static minute(count) {
    return this._regular(count, "minute");
  }

```

Reply:
I still think we should maybe just use the library I suggested, because they have all the rules for pluralization in English already coded in - despite not having the auto complete feature :D

I don't thing regular is right since, you have regular pluralization that are a little trickier, but still considered "regular", because there is a formula for it: 
(https://www.grammarly.com/blog/plural-nouns/#:~:text=Plural%20noun%20rules,-There%20are%20many&text=1%20To%20make%20regular%20nouns,add%20%E2%80%93s%20to%20the%20end.&text=2%20If%20the%20singular%20noun,end%20to%20make%20it%20plural.&text=3%20In%20some%20cases%2C%20singular,adding%20the%20%E2%80%93es%20for%20pluralization.)
![image](https://github.com/user-attachments/assets/059224cd-0d7c-48fa-aa32-f9ec418768d1)

Reply:
forget the regular.
but let's keep ours: I still think the API is nicer than their's. and we only have to add the words one at a time, as we're using them :)

Reply:
That's fair - we have our little own solution. ⭐